packages/api/src/schema/api-schema.yaml

```
@@ -167,17 +211,23 @@ components:
               - task.updated
               - task.completed
               - task.failed
+          example:
+            - stream.started
+            - stream.detection
```

The detection API is actually deprecated, let's give another example here
```suggestion
            - stream.idle
```